frontend/next.config.js

```diff
@@ -68,6 +68,9 @@ const nextConfig = {
       },
     ];
   },
+  env: {
+    auth_login_url: process.env.AUTH_LOGIN_URL,
```

Comment:
The same thing the `src/constants/environments.ts` is doing. Initially I had this as a `NEXT_PUBLIC_` var and that file says `home for all interpreted server side environment variables`. I'll move this over there, and update that comment since `NEXT_PUBLIC` files are defined there anyway. Good catch, thanks.